clustercontroller-reindexer/src/main/java/ai/vespa/reindexing/ReindexingMetrics.java

```diff
@@ -0,0 +1,47 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package ai.vespa.reindexing;
+
+import com.yahoo.documentapi.ProgressToken;
+import com.yahoo.jdisc.Metric;
+
+import java.time.Clock;
+import java.util.Map;
+
+import static ai.vespa.reindexing.Reindexing.State.SUCCESSFUL;
+
+/**
+ * Metrics for reindexing in a content cluster.
+ *
+ * @author jonmv
+ */
+class ReindexingMetrics {
+
+    private final Metric metric;
+    private final String cluster;
+
+    ReindexingMetrics(Metric metric, String cluster) {
+        this.metric = metric;
+        this.cluster = cluster;
+    }
+
+    void dump(Reindexing reindexing) {
+        reindexing.status().forEach((type, status) -> {
+            metric.set("reindexing.percent.done",
```

Comment:
Consider having "percent" removed from metric name, and instead represent it as a value between 0 and 1.

Reply:
Also, we should discuss the metric names in an architect meeting at some time.

Reply:
Yes, that’s the intention :) Feel free to reach out early with suggestions.